controllers/component_build_controller_pac.go

```diff
@@ -500,12 +501,17 @@ func (r *ComponentBuildReconciler) generatePaCPipelineRunConfigs(ctx context.Con
 	if err != nil {
 		return nil, nil, err
 	}
+	pipelineName, pipelineBundle, err := getNameAndBundle(pipelineRef)
+	if err != nil {
+		// TODO: Log the error? Wrap the error and return it? Record an event?
```

Comment:
For this specific case, I think we should just return permanent error, so the error will be visible in the build status annotation (and so on UI).

Reply:
Works for me. I'll try to test this in quickcluster to see if the logs will have enough details

Reply:
They do, nice.

```
Pipelines as Code provision for the Component failed {"controller": "component", "controllerGroup": "appstudio.redhat.com", "controllerKind": "Component", "Component": {"name":"devfile-sample-python-basic","namespace":"default"}, "namespace": "default", "name": "devfile-sample-python-basic", "reconcileID": "3c66bec3-5ec2-4736-9630-cb259eb93d7c", "error": "unsupported Tekton resolver \"git\""}
```
```
simple build submition for the Component failed {"controller": "component", "controllerGroup": "appstudio.redhat.com", "controllerKind": "Component", "Component": {"name":"devfile-sample-python-basic","namespace":"default"}, "namespace": "default", "name": "devfile-sample-python-basic", "reconcileID": "d590f08d-8187-46c6-9fe5-a1b118274fe0", "error": "unsupported Tekton resolver \"git\""}
```